### ISSUE TITLE
(low-priority) User session: Save the last step the user was at and restore it when user shuts down Godot

### ISSUE THREAD

**Issue by NathanLovato** [GDQuest/GDTour#54]
We have a user who closed the entire Godot editor halfway through a tour and expected the system to just reload and reopen at the step they left off.

This is a low priority task but it'd be ideal to save and restore the last step the user was on. The user's last step probably has to be serialized on each step change.

The main difficulty in this task is ensuring that taking the user back to the last step is 100% reliable and puts the bubble and UI in a perfect state. So, it depends on #1 or something similar to ensure we can put the editor in a specific state.
